### DISCUSSION TITLE
Windows are slow/laggy unless in fullscreen

### DISCUSSION THREAD

**Discussion by lupelagic** [i3/i3#5494] [Q&A] [answered]
I have a Thinkpad T16, and run Linux Mint with i3 gaps. I've noticed that any windows I have open, particularly browsers playing video, are incredibly slow unless I am in fullscreen. I've tested this with other applications and terminal processes and they're all slow or struggle to load in, even if they're not particularly demanding in terms of CPU usage. Not sure if I need to include any more info, but I would be happy to if needed! Any help would be appreciated.

**Comment by orestisfl** [comment, marked as answer]
Are you using a compositor like picom?

**Reply by lupelagic** [reply]
I don't think so? I'm not all too sure how to find out, but I didn't install one when first setting up afaik

edit: I've experimented with picom and compton, which both fix the issue! Problem is, both of them cause screen tearing that I can't quite sort out lol. I'll keep working on it
